blog/Website/2015-02-18_FreshRSS_gets_a_new_website.md

```diff
@@ -0,0 +1,6 @@
+Title: FreshRSS gets a new website
+Date: 2015-02-18
+
+We have worked a lot to improve community processes since last weeks. Today we are opening a new section on the website: a blog!
+
+TODO: this article needs to be finished!
```

Comment:
Yep, this one. I was not inspired today and I prefer to write article in a dedicated PR. Now I think about it, it would be great to delete this file too before merging this PR.